apps/pay/app/[username]/layout.tsx

```diff
@@ -27,10 +27,13 @@ type Props = {
 export default async function UsernameLayout({ children, params }: Props) {
   let response: ApolloQueryResult<AccountDefaultWalletsQuery> | { errorMessage: string }
   try {
-    response = await getClient().query<AccountDefaultWalletsQuery>({
-      query: AccountDefaultWalletsDocument,
-      variables: { username: params.username },
-    })
+    response = await apollo
+      .unauthorized()
```

Comment:
```suggestion
      .unauthenticated()
```

the correct terminology here is authenticated. 

its important to know the different between authentication and authorization (but sometimes I mess it myself as well)